Add `Touhou Project Mania Cup 4: Registrations are Now Open!` news post (#12576)
ppy/osu-wiki: news/2024/2024-12-22-touhou-project-mania-cup-4th-registrations-now-open.md

---
layout: post
title: "Touhou Project Mania Cup 4th: Registrations Now Open!"
date: 2024-12-22 00:00:00 +0000
---

![](/wiki/Tournaments/TMC/4th/image/banner@2x.png)

With the celebration of the Touhou Project's 30th birthday, the Touhou Project Mania Cup is returning for its 4th iteration, bringing tons of improvement in competitiveness and player enjoyment!

The [Touhou Project Mania Cup 4th](https://osu.ppy.sh/community/forums/topics/2015815), or THMC 4 for short, is a team-based osu!mania 4K tournament exclusively featuring Touhou-related songs. This tournament aims to provide top-tier competition to the tournament scene.

We've had 80 teams [last year](/wiki/Tournaments/TMC/3rd), where team **Bloxy cola** reached the top with an undefeated performance. [This year](/wiki/Tournaments/TMC/4th), with more teams joining the competition, which team will rise to the challenge and become the best? 

## What's new

> - **A Group Stage will be introduced in this year**, featuring 16 groups of 4 teams each. This means that we will have **64 teams** in the main event.

Reviewer: ```suggestion
- **A Group Stage will be introduced this year**, featuring 16 groups of 4 teams each. This means that we will have **64 teams** in the main event.
```

>   - If there are more than 70 teams participating in the Qualifier, the Group Stage will use the [split double-elimination format](/wiki/Tournaments/TMC/4th#Group-Stage-&-Playoffs).

Reviewer: ```suggestion
  - If there are fewer than 70 teams playing in the Qualifier, the Group Stage will use the standard double-elimination format, featuring 8 groups of 4 teams each.
```

>   - If there are fewer than 70 teams participating in the Qualifier, the Group Stage will use the standard double-elimination format.

Reviewer: ```suggestion
  - If there are 70-140 teams playing the Qualifier, the Group Stage will use the split [double-elimination format](/wiki/Tournaments/TMC/4th#Group-Stage-&-Playoffs), featuring 16 groups of 4 teams each. This means that we will have **64 teams** in the main event.
  - More large-scale announcements will be made if more than 140 teams played the Qualifier!
```

![Group format](/wiki/Tournaments/TMC/4th/image/group-stage-bracket.png)

>   - **Each team will have 2 bans in the group stage**, providing extra strategic options throughout the different weeks.

Reviewer: ```suggestion
  - **Each team will have 2 bans for matches played using the group stage mappools**, providing extra strategic options throughout the different weeks.
```

- **A special series of "X" picks will apply for RC and LN slots**, giving players more options to pick harder charts.

> - Two teams from each group will advance, with a total of 32 teams competing in a standard double-elimination bracket for the championship.

Reviewer: ```suggestion
- Two teams from each group will advance; all advanced teams will compete in a standard double-elimination bracket for the championship.
```

## Custom maps

> With nearly **50 custom mappers** contributing, this year we are bringing you mappools that are **almost entirely custom-made** with maps ranging from the lower end of the difficulty spectrum to the very top. We hope this will not only enhance our Touhou journey, but also contribute to the osu! community by adding more Touhou maps!

Reviewer: ```suggestion
With nearly **50 custom mappers** contributing, this year we are bringing you mappools that are **almost entirely custom-made** with maps ranging from the lower end of the difficulty spectrum to the very top. We hope this will not only enhance our Touhou journey but also contribute to the osu! community by adding more Touhou maps!
```

You can check out [Supa7onyz](https://osu.ppy.sh/beatmaps/artists/435)'s brilliant Grand Finals custom tiebreaker song from last year as an example:

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="//player.bilibili.com/player.html?isOutside=true&aid=487714748&bvid=BV1sN411D7RV&cid=1176362169&p=1" frameborder="0" allowfullscreen></iframe>
</div>

## Tournament schedule

| Event | Dates |
| --: | :-- |
| Registration phase | Dec 22, 2024–Jan 12, 2025 (23:59 UTC) |
| Qualifier showcase | Jan 19 (12:00 UTC) |
| Qualifier stage | Jan 20–Jan 28 |
| Group stage Week 1 | Jan 28–Feb 9 |
| Group stage Week 2 | Feb 14–Feb 16 |
| Round of 32 | Feb 21–Feb 23 |
| Round of 16 | Feb 28–Mar 2 |
| Quarterfinals | Mar 7–Mar 9 |
| Semifinals | Mar 14–Mar 16 |
| Finals | Mar 21–Mar 23 |
| Grand Finals | Mar 28–Mar 30 |

## How do I register?

Simply click [here](https://docs.google.com/forms/d/1tl-4NeWZIDI10-1Mkja8lT3DHEpIUB_PK2uqAHptVj8/) and fill in the team registration form!

> In order to participate in this tournament, you will need to ensure that...

Reviewer: ```suggestion
In order to participate in this tournament, you must make sure that...
```
